corehq/apps/app_manager/id_strings.py

```diff
@@ -90,6 +90,16 @@ def detail_column_header_locale(module, detail_type, column):
     )
 
 
+@pattern('m%d.%s.%s_%s_%d.header')
+def invisible_column_header_locale(module, detail_type, column):
+    return u"m{module.id}.{detail_type}.{d.model}_{field}_{d_id}.header".format(
```

Comment:
Ah, right the `field` would be different from any of the other ones. Since it's the same is there any point in having a separate method?